commands/host.vegman

```diff
@@ -303,8 +289,8 @@ function cmd_config_default {
     cancel) config_default_cancel $yes;;
     status) config_default_status;;
     set)    config_default_set $yes;;
-    *) echo "Unsupported parameter. Use --status, --cancel or --yes to
-approve";;
+    *) echo "Unsupported parameter. Use --status, \
+             --cancel or --yes to approve";;
```

Comment:
Wrong. This will pring a long string of spaces between "--status," and "--cancel".
You need to close the quote before `\` and then reopen it after indentaion. See my previous comment.

Also, why are there 3 commits now? I expected 2: one for the config_default, and one for help refactoring.

Reply:
Oh, and I totally missed that! Andrey was correct. You can never get to this line at all!
You've already directly set `action` in the above loop to one of the three values.
I'd remove this `case` completely and just call `config_default_${action} "${yes}"` here.